### ISSUE TITLE
Static files have invalid links in manifest

### ISSUE THREAD

**Issue by PatrickSachs** [hanford/next-offline#101]
Related: #44 - Although phrased similar, I don't think this is related to #84 

I'm a bit sorry for opening so many tickets/PRs within a short timeframe, but I really appreciate your effort and time put into this small library.

- **Issue and steps to reproduce.**

Currently all assets always always prefixed with `/_next/`. This works well for "normal" next.js files, but not for static assets that are discovered using the glob options in the `/static` folder and then served in the `/static` folder (without `_next`!)

**- Versions.**
   
```json
{
  "next": "^7.0.2",
  "next-offline": "^3.3.1"
}
```

**- Actual/Screenshots.**

![screenshot from 2018-12-13 18-16-32](https://user-images.githubusercontent.com/7840502/49955656-a7342f80-ff03-11e8-857e-c41d994aa2ab.png)

Requesting these resources using `/_next/static/content/system/default-avatar.png` results in a 404 error since next serves them from `/static/content/system/default-avatar.png`. This not a feature specific to my application, this is a built in Next feature.

https://github.com/zeit/next.js/#static-file-serving-eg-images

**- Expected.**

![screenshot from 2018-12-13 18-23-53](https://user-images.githubusercontent.com/7840502/49955924-4a854480-ff04-11e8-8e5d-6a6cbb5f77af.png)

**- Link to your project.**

Private repository

**- Browser name and version.**

```
User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.80 Safari/537.36

patrick@PATRICK:~/Development/helios/git$ node --version
v10.13.0
patrick@PATRICK:~/Development/helios/git$ npm --version
6.4.1
```

**- Ideas**

I did not find a way to distinguish between next.js generated assets and assets in the /static directory in the manifest generated by workbox (result of `getOriginalManifest` in the code), so I'm not sure on how to approach this.

An idea would be to manually check if the files exist in the /static directory, but this seems like a rather dirty implementation/fix.

Another idea would be not to use the glob setting by provided by workbox and instead discover the assets in `/static` on our own.

**Comment by PatrickSachs** [comment]
Hey, I didn't have the time yesterday, but here you go - https://github.com/PatrickSachs/next-offline-example

Thanks for the time. My issue seems to be related(again!) to not using `generateSw: true`, which then uses the InjectManifest mode in the background.

According to a comment in the workbox repository this mode is the one that should be used if you just want to have the manifest generated for you, but want to handle everything else yourself.

**Comment by k-prashant** [comment]
I don't know if my issue is related to this, so instead of opening new one, adding it here.

`
// next.config.js

dontAutoRegisterSw: true,
generateSw: false,
generateInDevMode: true,

workboxOpts: {

  swSrc: './custom-sw.js',
  swDest: 'custom-sw.js',

  importWorkboxFrom: "disabled",
  globDirectory: '.next',
  globPatterns: [
    '../static/**/*',
  ],
}
`
I have set generateSw as false to use injectmanifest options. I had to use globDirectory and globPatterns different to get precache manifest.



`

// generated .next/custom-sw.js

self.__precacheManifest = [
  {
    "url": "/_next/../static/img/icon_640x1136.png"
  },
  {
    "url": "/_next/static/chunks/commons.1cab0315ed9859ff36d1.js"
  },
  {
    "url": "/_next/static/runtime/main-1d96240c41330024a8ea.js"
  },
  {
    "url": "/_next/static/kXiJDmzxtIvmdNFowXros/pages/_app.js"
  },
  {
    "url": "/_next/static/kXiJDmzxtIvmdNFowXros/pages/_error.js"
  },
  {
    "url": "/_next/../static/img/apple-touch-icon-57x57.png"
  },
  {
    "url": "/_next/../static/img/loader.svg"
  },
  {
    "url": "/_next/../static/fonts/Roboto-Medium-webfont.ttf"
  },
  {
    "url": "/_next/../static/css/style.css"
  }
];

importScripts();

importScripts("https://storage.googleapis.com/workbox-cdn/releases/3.6.3/workbox-sw.js");

workbox.precaching.precacheAndRoute(self.__precacheManifest || []);



workbox.skipWaiting()
workbox.clientsClaim()


workbox.precaching.suppressWarnings();
`


 If default options for globDirectory and globPatterns were used, I would have got below sw.

`
importScripts("precache-manifest.94ae417c04f1b889b9e688a195269904.js");

importScripts("https://storage.googleapis.com/workbox-cdn/releases/3.6.3/workbox-sw.js");

workbox.precaching.precacheAndRoute(self.__precacheManifest || []);



workbox.skipWaiting()
workbox.clientsClaim()


workbox.precaching.suppressWarnings();
`

So when I use this sw, I get an 404 not found for precache-manifest-xxx.js


Which sw should I use or do we have to make changes in next.config.js?
